doc/content/source/indicators/ElementSizeIndicator.md

```diff
@@ -0,0 +1,32 @@
+# ElementSizeIndicator
+
+!syntax description /Adaptivity/Indicators/ElementSizeIndicator
+
+## Description
+
+This indicator takes `element_size` MaterialProperty as an input and directly adapts
+the mesh element size to be equal to the `element_size`. Enough number of `initial_steps`
```

Comment:
Does the MaterialPropertyName have to be element_size? From the code, it looks like it can be anything.